Bottom: 621fa0d8d74cc2ac93a98a0a689e1854cbc22084
Top:    de795530562064436462399dfe158865988794ad
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-28 18:10:48 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 2ce49bf..16ea31a 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1725,7 +1725,7 @@ static void usage(void)
 		"	-f --foreground do not fork into background\n"
 		"	-r --random-multimount-selection\n"
 		"			use ramdom replicated server selection\n"
-		"	-m --dumpmaps [<map type> <map name]"
+		"	-m --dumpmaps [<map type> <map name]\n"
 		"			dump automounter maps and exit\n"
 		"	-n --negative-timeout n\n"
 		"			set the timeout for failed key lookups.\n"
